README.md

```diff
@@ -99,9 +98,9 @@ All possible features are listed below:
 
 ##### Installation steps
 - Create directory `C:\opencv`.
-- Copy files from `.windows` folder there.
+- Copy `.git` and `.windows` folders there (you can run them from the `cv-rs` directory itself, but you may encounter an error that paths are too long)
```

Comment:
Why is `.git` folder needed for installation?

Reply:
Because of submodule restoration.

I think opencv installation probably should be moved into separated crate. And `cv-rs` should just have a link to the repo.

Reply:
Yes, `opencv installation` is better left separate. But what do you mean by "separate crate"?

Reply:
Sorry, I meant separate repository ;)